website/src/types/model.ts

```diff
@@ -15,7 +15,7 @@ export type Page = {
 };
 
 /**
- * アウトライン情報
+ * Outline information.
```

Comment:
```suggestion
 * An element in the outline.
```

(adapted from the original `docs/src/model.rs`: _An element in the "On This Page" outline_)